client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -95,6 +95,7 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 		handleToggle = name => () => {
 			this.props.trackEvent( `Toggled ${ name }` );
 			this.props.updateFields( { [ name ]: ! this.props.fields[ name ] } );
+			this.props.markChanged();
```

Comment:
Can we be smarter about this if a user checks, then unchecks (hence resetting to original value) a checkbox?